scipy/interpolate/fitpack2.py

```diff
@@ -1031,36 +1041,42 @@ def _from_tck(cls, tck):
 
     def ev(self, xi, yi, dx=0, dy=0):
         """
-        Evaluate the spline at points
+        Evaluate the spline at the points `xi` and `yi`.
```

Comment:
The description bellow is more accurate. Just missing a point but it is accurate.

Reply:
Shouldn't be it `dxi` and not `dx`?

Reply:
I did not check the code, but just reading the change would give me the impression that x and y could be 2 points.